internal/oc/span.go

```diff
@@ -15,3 +20,29 @@ func SetSpanStatus(span *trace.Span, err error) {
 	}
 	span.SetStatus(status)
 }
+
+// StartSpan wraps go.opencensus.io/oc.StartSpan, but, if the span is sampling,
+// adds a log entry to the context that points to the newly created span.
+func StartSpan(ctx context.Context, name string, o ...trace.StartOption) (context.Context, *trace.Span) {
+	ctx, s := trace.StartSpan(ctx, name, o...)
+	return update(ctx, s)
+}
+
+// StartSpanWithRemoteParent wraps go.opencensus.io/oc.StartSpanWithRemoteParent.
```

Comment:
either doc is inaccurate or the implementation?

Reply:
it should be `trace.StartSpanWithRemoteParent`.
I think i did a mass search/replace from `trace.StartSpan` to `oc.StartSpan`, and this was a casualty